apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-snap-scroll-example.component.html

```diff
@@ -0,0 +1,141 @@
+<fdp-dynamic-page background="transparent" size="medium" ariaLabel="Example Dynamic Page">
+    <fdp-dynamic-page-title
+        title="Balenciaga Tripple S Trainers"
+        subtitle="Oversized multimaterial sneakers with quilted effect"
+    >
+        <!-- breadcrumb content -->
+        <fd-breadcrumb>
+            <fd-breadcrumb-item>
+                <a fd-breadcrumb-link [attr.href]="'#'">Men</a>
+            </fd-breadcrumb-item>
+            <fd-breadcrumb-item>
+                <a fd-breadcrumb-link [attr.href]="'#'">Shoes</a>
+            </fd-breadcrumb-item>
+            <fd-breadcrumb-item>
+                <a fd-breadcrumb-link [attr.href]="'#'">Running Shoes</a>
+            </fd-breadcrumb-item>
+            <fd-breadcrumb-item>
+                <span>Jogging</span>
+            </fd-breadcrumb-item>
+        </fd-breadcrumb>
+
+        <fdp-dynamic-page-key-info>
+            <!-- key information content -->
+            LAST DAY OF SALE
+        </fdp-dynamic-page-key-info>
+        <fdp-dynamic-page-global-actions>
+            <!-- global actions -->
+            <fd-toolbar [fdType]="'transparent'" [clearBorder]="true">
```

Comment:
[] binding not required. Same in other places as well. For more details, other component examples will help

Reply:
removed binding.. had lifted examples from core and didn't notice. Thanks for noticing!